molgenis-core-ui/src/test/javascript/modules/rest-client/rsql/transformerTest.js

```diff
@@ -0,0 +1,219 @@
+import {parser} from "rest-client/rsql";
+import {
+    toRange,
+    toRangeLine,
+    groupBySelector,
+    toText,
+    toSimpleRef,
+    toComplexRef,
+    toComplexLine
+} from "rest-client/rsql/transformer";
+import test from "tape";
+
+test('Test that a single comparison gets mapped', assert => {
+    const actual = groupBySelector(parser.parse("xbool==false"))
+    const expected = {
+        "xbool": {
+            "selector": "xbool",
```

Comment:
Shouldnt there be a type here as well? like RANGE or SIMPLE_REF
Else I still need to manually check for types and / or operators